flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
this is disappointing.

Reply:
I can't do anything with this. Pyfa releases are built for x86_64. Other architectures have never been claimed to support.